src/SpendPermissionManager.sol

```diff
@@ -468,29 +471,29 @@ contract SpendPermissionManager is EIP712 {
         _transferFrom(spendPermission.token, spendPermission.account, spendPermission.spender, value);
     }
 
-    /// @notice Check if a spend permission is revoked.
+    /// @notice Get if a spend permission is approved.
```

Comment:
not officially documented, but our use of "check" is paired with a conditional and potential revert, so unified with our existing convention of "get" for other view functions.